src/service.rs

```diff
@@ -142,6 +146,21 @@ impl Service {
                         }
                     }
                     Ok(None) => {
+                        if last_updated_time.elapsed() > Duration::from_secs(60)
+                            && last_warning_time.elapsed() > Duration::from_secs(60)
+                        {
+                            if let Ok(ckb_tip_header) = get_tip_header(&rpc_client).await {
+                                error!(
+                                    "ckb-indexer has been {}s since the last update, ckb.tip_number = {}, ckb.tip_hash = {:#x}, indexer.tip_number = {}, indexer.tip_hash = {:#x}",
```

Comment:
this error message `ckb-indexer has been 60s since the last update, ` doesn't read like a complete sentence

Reply:
Force push 5b467ca0 updates:
* rewrite "ckb-indexer has been 60s since the last update" to "it has been 60s since the last update"